assembl/static2/js/app/pages/administration.jsx

```diff
@@ -109,6 +118,17 @@ class Administration extends React.Component {
     this.props.updateResourcesCenterPage(filteredResourcesCenter.resourcesCenter);
   }
 
+  putVoteSessionInStore(voteSession) {
+    const filteredVoteSession = filter(VoteSessionQuery, { voteSession: voteSession });
+    const voteSessionForStore = {
+      ...filteredVoteSession.voteSession,
+      instructionsSectionContentEntries: filteredVoteSession.voteSession.instructionsSectionContentEntries
```

Comment:
it may be nice to move this `if` in the `convertEntriesToRawContentState` helper ?